### PR TITLE
add recommendation docs

### PR DESCRIPTION
**Pull request by mrandi**
add documentation

### CODE REVIEW

**Comment by mrandi** [comment]
👍

File: source/includes/_catalog-api.md [hunk 1]
```diff
@@ -152,3 +152,76 @@ Parameter | Type | Description | Required
...
+
+### Request
+
+`GET /api/articles/{config-sku}/recommendations?sales_channel={sales-channel}&client_id={client-id}`
+
+Parameter | Type | Description | Required
+--------- | ---- | ----------- | --------
+config_sku | Path | The config SKU of the article. | Yes
+client_id | Query | Client ID for the client making the request. Used to track the origin of the request. | Yes
+sales_channel | Query | Sales Channel under which the article is being sold. The sales channel gives us the country and currency to show the price. | Yes
```
Comment (Sangdol) [inline source/includes/_catalog-api.md]: Now we don't take `sales_channel` as a request parameter, but we take it as a header. I think we need to update here and also the part of `getArticle` API.

Comment (pc-alves) [inline source/includes/_catalog-api.md]: Valid point. But that needs to be fixed across the whole Catalog-API section. Meaning the other 2 endpoints.

Comment (mrandi) [inline source/includes/_catalog-api.md]: thx, now is done

**Comment by Sangdol** [comment]
👍

**Comment by pc-alves** [comment]
👍

**Comment by pc-alves** [comment]
Needs to be aligned after changes to the API on the recommendation endpoint.

**Comment by Sangdol** [comment]
👍

**Comment by mrandi** [comment]
👍

**Pull request merged**